src/renderer/features/search/components/library-command-item.tsx

```diff
@@ -169,19 +168,6 @@ export const LibraryCommandItem = ({
                 >
                     <RiPlayFill />
                 </Button>
-                <Button
-                    compact
-                    disabled={disabled}
-                    onClick={(e) => handlePlay(e, id, Play.SHUFFLE)}
-                    size="md"
-                    tooltip={{
-                        label: t('player.play', { postProcess: 'sentenceCase' }),
-                        openDelay: 500,
-                    }}
-                    variant="default"
-                >
-                    <RiShuffleFill />
-                </Button>
```

Comment:
You don't need to remove this.

Just do a filter it out for `songs` only. Like `itemType !== LibraryItem.SONG ...`